app/src/main/java/com/example/parkingsystem/mvp/presenter/ParkingSizeDialogPresenter.kt

```diff
@@ -0,0 +1,27 @@
+package com.example.parkingsystem.mvp.presenter
+
+import com.example.parkingsystem.listener.ParkingDialogListener
+import com.example.parkingsystem.mvp.contract.ParkingDialogContract
+import com.example.parkingsystem.util.Constants
+
+class ParkingSizeDialogPresenter(private val view: ParkingDialogContract.ParkingSizeDialogFragmentView) :
+    ParkingDialogContract.ParkingSizeDialogFragmentPresenter {
+
+    override fun onButtonDialogFragmentOkPressed(listener: ParkingDialogListener) {
+        val parkingSize = view.getParkingLots()
+
+        if (parkingSize.isNotEmpty() && parkingSize.toInt() < Constants.PARKING_SIZE_MAX) {
+            val intParkingLots = parkingSize.toInt()
```

Comment:
the line **parkingSize.toInt()** is repeated so you can put this variable outside the if

Reply:
Done